src/components/Incidents/IncidentCreate/index.tsx

```diff
@@ -255,34 +258,35 @@ export function IncidentCreate({
           <p className="text-red-600 text-sm">{errors.tracking?.message}</p>
         </div> */}
 
-        <div className="mb-4">
-          {/* have a look at this */}
-          <ReactSelectDropdown
-            control={control}
-            label="Severity"
-            name="severity"
-            className="w-full"
-            items={severityItems}
-            containerClassName="flex flex-col space-y-1"
-            labelClass="block text-sm font-bold text-gray-700 mb-2"
-            value={severity}
-          />
-          <p className="text-red-600 text-sm">{errors.severity?.message}</p>
-        </div>
-        <div className="mb-4">
-          <ReactSelectDropdown
-            control={control}
-            label="Status"
-            name="status"
-            className="w-full"
-            items={incidentStatusItems}
-            labelClass="block text-sm font-bold text-gray-700 mb-2"
-            containerClassName="flex flex-col space-y-1"
-            value={status}
-          />
-          <p className="text-red-600 text-sm">{errors.status?.message}</p>
+          <div className="mb-4">
+            {/* have a look at this */}
+            <ReactSelectDropdown
+              control={control}
+              label="Severity"
+              name="severity"
+              className="w-full"
+              items={severityItems}
+              containerClassName="flex flex-col space-y-1"
+              labelClass="block text-sm font-bold text-gray-700 mb-2"
+              value={severity}
+            />
+            <p className="text-red-600 text-sm">{errors.severity?.message}</p>
+          </div>
+          <div className="mb-4">
+            <ReactSelectDropdown
+              control={control}
+              label="Status"
+              name="status"
+              className="w-full"
+              items={incidentStatusItems}
+              labelClass="block text-sm font-bold text-gray-700 mb-2"
+              containerClassName="flex flex-col space-y-1"
+              value={status}
+            />
+            <p className="text-red-600 text-sm">{errors.status?.message}</p>
+          </div>
         </div>
-        <div className="flex justify-end">
+        <div className="flex justify-end bg-gray-100 px-5 py-4 w-full rounded">
```

Comment:
This button should be the same style as the others and should use the <Button> component rather than the button dom element